core/node/libp2p/rcmgr_defaults.go

```diff
@@ -56,10 +57,21 @@ func createDefaultLimitConfig(cfg config.SwarmConfig) (rcmgr.LimitConfig, error)
 
 	numFD := cfg.ResourceMgr.MaxFileDescriptors.WithDefault(int64(fd.GetNumFDs()) / 2)
 
+	// We want to see this message on startup, that's why we are using fmt instead of log.
+	fmt.Printf(`
+Computing default go-libp2p Resource Manager limits based on: 
+    - 'Swarm.ResourceMgr.MaxMemory': %q
+    - 'Swarm.ResourceMgr.MaxFileDescriptors': %d
+
+Applying any user-supplied overrides on top.
+Run 'ipfs swarm limit all' to see the resulting limits.
+
+`, maxMemoryString, numFD)
+
 	scalingLimitConfig := rcmgr.ScalingLimitConfig{
 		SystemBaseLimit: rcmgr.BaseLimit{
-			Memory: int64(maxMemory),
-			FD:     int(numFD),
+			Memory: rcmgr.DefaultLimits.SystemBaseLimit.Memory,
+			FD:     rcmgr.DefaultLimits.SystemBaseLimit.FD,
```

Comment:
I think we should leave this as is, but then go and fix

```
SystemLimitIncrease: rcmgr.BaseLimitIncrease{
			Memory:  0,
			FDFraction: 0
```

That way we fix the value and don't do any incrementing.  Thats like what we're doing with Conns.  We set it to big enough and then increase by 0.

Reply:
good catch. Changed.